spec/factories/models/forms.rb

```diff
@@ -37,5 +38,9 @@
       support_url { Faker::Internet.url(host: "gov.uk") }
       support_url_text { Faker::Lorem.sentence(word_count: 1, random_words_to_add: 4) }
     end
+
+    trait :from_other_org do
+      org { "another-org" }
+    end
```

Comment:
Why do we need this trait? if we need it is to set this in a test, it is possible to just pass it in when we build it

Reply:
Didn't realise we could do that, great!